azure/deploy-to-azure/parent_template.json

```diff
@@ -72,16 +72,16 @@
       }
     },
     "diagnosticSettingName": {
-        "type": "string",
-        "defaultValue": "[concat('datadog-activity-logs-diagnostic-setting-', uniqueString(newGuid()))]",
-        "metadata": {
-            "description": "The name of the diagnostic setting if sending Activity Logs"
-        }
+      "type": "string",
+      "defaultValue": "[concat('datadog-activity-logs-diagnostic-setting-', uniqueString(newGuid()))]",
+      "metadata": {
+        "description": "The name of the diagnostic setting if sending Activity Logs"
+      }
     }
   },
   "variables": {
     "eventHubTemplateLink": "https://raw.githubusercontent.com/DataDog/datadog-serverless-functions/master/azure/deploy-to-azure/event_hub.json",
-    "functionAppTemplateLink": "https://raw.githubusercontent.com/DataDog/datadog-serverless-functions/master/azure/deploy-to-azure/function_template.json",
+    "functionAppTemplateLink": "https://raw.githubusercontent.com/DataDog/datadog-serverless-functions/tree/rebecca.parsons/upgrade_azure_forwarder_node_20/azure/deploy-to-azure/function_template.json",
```

Comment:
"https://raw.githubusercontent.com/DataDog/datadog-serverless-functions/rebecca.parsons/upgrade_azure_forwarder_node_20/azure/deploy-to-azure/function_template.json"